content/en/1-0/vcjob.md

```diff
@@ -308,20 +308,20 @@ spec:
 ## Note
 ### Frameworks Supported
 Volcano support almost all mainstream computing frameworks including:
-
-1. tensorflow
-2. pytorch
-3. mindspore
-4. PaddlePaddle
-5. spark
-6. flink
-7. openMPI
-8. horovod
-9. mxnet
-10. kubeflow
-11. argo
-12. kubeGene
+* tensorflow
```

Comment:
Please review this PR again for too much unsynchronized code between your local project and master. Keep the changes  of master to your local